pkg/controller/scyllacluster/controller.go

```diff
@@ -41,6 +41,8 @@ const (
 	// maxSyncDuration enforces preemption. Do not raise the value! Controllers shouldn't actively wait,
 	// but rather use the queue.
 	maxSyncDuration = 30 * time.Second
+
+	artificialDelayForCachesToCatchUp = 10 * time.Second
```

Comment:
I think reconcilation of three rack cluster would extend the maxSyncDuration due sleeps. Let's account for these sleeps in sync timeout calculation

Reply:
yep, I'll aggregate then to one in a follow up

Reply:
https://github.com/scylladb/scylla-operator/pull/685